TestTraceflow/testTraceflowEgress/egressFromRemoteNode failing consistently on dual-stack testbed

**Describe the bug**
The `TestTraceflow/testTraceflowEgress/egressFromRemoteNode` e2e test is failing consistently on the dual-stack testbed.
```
=== RUN   TestTraceflow/testTraceflowEgress/egressFromRemoteNode
    traceflow_test.go:2252: Latest Traceflow status: {Running  2022-12-20 21:44:51 +0000 UTC 7 [{antrea-ipv6-9-1  1671572693 [{SpoofGuard  Forwarded     0    } {Egress  ForwardedToEgressNode    egress-wgkqd 0    } {Forwarding Output Forwarded     0    }]}] <nil>}
    traceflow_test.go:2439: Error: Get Traceflow failed: timed out waiting for the condition
```

The test was recently added as part of https://github.com/antrea-io/antrea/pull/3926

--- a/pkg/agent/controller/traceflow/packetin.go
+++ b/pkg/agent/controller/traceflow/packetin.go
@@ -237,7 +237,8 @@ func (c *Controller) parsePacketIn(pktIn *ofctrl.PacketIn) (*crdv1alpha1.Tracefl
 	if tableID == openflow.L2ForwardingOutTable.GetID() {
 		ob := new(crdv1alpha1.Observation)
 		tunnelDstIP := ""
-		isIPv6 := c.nodeConfig.NodeIPv6Addr != nil
+		// decide according to packet.
+		isIPv6 := etherData.Ethertype == protocol.IPv6_MSG
 		if match := getMatchTunnelDstField(matchers, isIPv6); match != nil {
 			tunnelDstIP, err = getTunnelDstValue(match)
 			if err != nil {

